docs/distributed/DESIGN.md

```diff
@@ -106,6 +85,50 @@ Expected expansion
 > http://host4/export8
 ```
 
+A noticeable trait of this expansion is that it chooses unique hosts such that the erasure code is efficient across drives across hosts.
+
+- Choosing an erasure pack for the object is decided during `PutObject()`, object names are used to find the right erasure pack using the following pseudo code.
```

Comment:
It was called `erasure set` earlier, have we changed the name to `erasure pack` now?

Reply:
Well set or pack is not a hard technical term that we are religious about. Pack is perhaps something preferable, although set is mathematically correct. But you tell me..

Reply:
Probably just because I am familiar with `erasure set` it sounds natural/preferable to me..

Reply:
Okay let me keep it `erasure set`